recipes/bycycle/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "bycycle" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
It looks like the package doesn't include the `requirements.txt` in the source distribution, which is breaking the `setup.py`. I've submitted a PR to fix this upstream:

https://github.com/bycycle-tools/bycycle/pull/65

For now, you can base this on a release:

https://github.com/bycycle-tools/bycycle/releases
https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L17

Reply:
Thanks for the review.

This wasn't an issue for other packages I maintain (https://github.com/conda-forge/fooof-feedstock) and the recent (2-3 weeks ago) build for the linked feedstock was successful. This also doesn't seem to effect pypi installations. Is this due to a recent change in how conda-forge builds packages?

Reply:
If you try to install `bycycle` via `pip` without using the wheel (which is the default), you get an error:

```
$ pip install --no-binary=:all: bycycle

ERROR: Command errored out with exit status 1:
...
Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/ld/gx_9s3_n55g1dkhj59zjlxww0000gn/T/pip-install-nbu1vzpc/bycycle/setup.py", line 15, in <module>
        with open("requirements.txt") as requirements_file:
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
```

If you look at `fooof`, it does not try to read in a `requirements.txt` in its `setup.py`, which is why it works. Again, my recommendation is to switch from using the release on `pypi` to the github release which includes the `requirements.txt`. I think we would eventually want to switch back to the tarball on `pypi`, because it is much smaller than the github release.

Reply:
Thanks a ton for the explanation. It cleared up my confusion on this PR and on another feedstock.